azure-pipelines.yml

```diff
@@ -226,7 +226,7 @@ stages:
           organization: 'sonarsource'
           scannerMode: 'MSBuild'
           projectKey: 'sonaranalyzer-dotnet'
-          projectName: 'SonarAnalyzer .Net'
+          projectName: 'SonarAnalyzer .NET'
```

Comment:
I'm undecided about the space, if it should be there or not...

Reply:
We can use the same pattern we use for the scanner. `SonarAnalyzer for .Net`